### ISSUE TITLE
InternalsVisibleTo In Csproj Is Not Supported In CLI

### ISSUE THREAD

**Issue by warappa** [JosefPihrt/Roslynator#970]
**Product and Version Used**: 
Roslynator.DotNet.Cli: **3.3**
`dotnet --version`: **7.0.100-rc.2.22477.23**
Target framework: **net6.0**

**Steps to Reproduce**:
Create solution with 2 projects
* BaseProject
* TestProject

Add an internal class like `InternalClass` to BaseProject.

Add the following code to BaseProject:
```xml
<ItemGroup>
  <InternalsVisibleTo Include="TestProject" />
</ItemGroup>
```
Use `InternalClass` in TestProject.

Compile with Visual Studio - works.

Run `roslynator fix solution.sln`

**Actual Behavior**:
Error **CS0122** is raised for `InternalClass` in TestProject.

**Expected Behavior**:
No error.

**Comment by JosefPihrt** [comment]
Hi,

I think that the problem is that you target .NET 7 which has not been released yet (the stable version) thus it is not supported by Roslynator. If you target .NET 6 the command will run without any errors.

When the stable version of .NET 7.0 is released it will be supported by Roslynator.

**Comment by warappa** [comment]
This seems to be fixed.
